src/ast/nodes/AssignmentExpression.ts

```diff
@@ -52,29 +52,46 @@ export default class AssignmentExpression extends NodeBase {
 		this.left.render(code, options);
 		this.right.render(code, options);
 		if (options.format === 'system') {
-			if (this.left.variable && this.left.variable.exportName) {
+			const exportNames = this.left.variable?.safeExportName
+				? [this.left.variable?.safeExportName]
+				: this.left.variable?.exportName;
+			const _ = options.compact ? '' : ' ';
+			if (exportNames) {
 				const operatorPos = findFirstOccurrenceOutsideComment(
 					code.original,
 					this.operator,
 					this.left.end
 				);
 				const operation =
-					this.operator.length > 1
-						? ` ${this.left.variable.exportName} ${this.operator.slice(0, -1)}`
-						: '';
-				code.overwrite(
-					operatorPos,
-					operatorPos + this.operator.length,
-					`= exports('${this.left.variable.exportName}',${operation}`
-				);
-				code.appendLeft(this.right.end, `)`);
+					this.operator.length > 1 ? ` ${exportNames[0]} ${this.operator.slice(0, -1)}` : '';
```

Comment:
This is from the existing source, but should this `exportNames[0]` be `this.left.variable.getName()`?